requirements/base.txt

```diff
@@ -19,7 +19,7 @@ tenacity>=6.2,<8
 django<4
 channels>=2.4.0,<3
 asgiref>=3.2.0,<3.2.2  # depends on our channels version
-mysqlclient<2.0.3  # keep in sync with edx-platform
-jsonfield2>=3.0.3,<4   # keep in sync with edx-platform
-pyguacamole>=0.10
+mysqlclient<2.1.0  # keep in sync with edx-platform
+jsonfield>=3.1.0,<4   # keep in sync with edx-platform
```

Comment:
If this means we can no longer install on Lilac (because it uses a `jsonfield` earlier than 3.1.0), then we need a big bold warning in the changelog about that.

And if we're dropping Lilac compatibility, and can run future releases only Maple, are we sure we need `jsonfield` anymore at all? After all, Django [has a native JSONField, as of 3.1](https://docs.djangoproject.com/en/4.0/releases/3.1/#jsonfield-for-all-supported-database-backends). Does Open edX not use that?